src/relayFeeCalculator/chain-queries/ethereum.ts

```diff
@@ -67,18 +72,21 @@ export const SymbolMapping: { [symbol: string]: { address: string; decimals: num
   },
 };
 
-export const defaultAverageGas = 116006;
-
 export class EthereumQueries implements QueryInterface {
+  private spokePool: EthereumSpokePool;
+
   constructor(
-    public readonly provider: providers.Provider,
-    public readonly averageGas = defaultAverageGas,
-    readonly symbolMapping = SymbolMapping
-  ) {}
+    readonly provider: providers.Provider,
+    readonly symbolMapping = SymbolMapping,
+    readonly spokePoolAddress = "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381",
+    readonly usdcAddress = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    readonly simulatedRelayerAddress = "0x9A8f92a830A5cB89a3816e3D267CB7791c16b04D"
```

Comment:
This is perfect for testing!

Before we merge, we should make sure all the default addresses are moved to `0x893d0D70AD97717052E3AA8903D9615804167759` and that address has the right funding and approvals on all chains.

Reply:
Not a problem! Currently `0x893d0D70AD97717052E3AA8903D9615804167759` doesn't have the proper access on ETH mainnet & Polygon. I will work to set it properly

Reply:
Thanks for the help! Updated [here](https://github.com/across-protocol/sdk-v2/pull/64/commits/45df24b38a73b80e4d830afa74ce4572505ea1d5).